potodo/_github.py

```diff
@@ -36,12 +36,20 @@ def get_repo_name(repo_path: str) -> str:
 def get_reservation_list(repo_path: str) -> Mapping[str, str]:
     """Will get the repository name then request all the issues and put them in a dict
     """
-    # Gets the issues into a dict
-    issues = requests.get(
-        "https://api.github.com/repos/" + get_repo_name(repo_path) + "/issues"
-    ).json()
-    # Creates the issue dict with the name and the login
-    reservations: dict = {
-        issue["title"].split()[-1].lower(): issue["user"]["login"] for issue in issues
-    }
+
+    issues: list = []
+    next_url = "https://api.github.com/repos/" + get_repo_name(repo_path) + "/issues?state=open"
+    while next_url:
+        resp = requests.get(next_url)
+        issues.extend(resp.json())
+        next_url = resp.links.get("next", {}).get("url")
+
+    reservations = {}
+
+    for issue in issues:
+        # Maybe find a better way for not using python 3.8 ?
+        if re.search(r'\w*/\w*\.po', issue['title']):
+            yes = re.search(r'\w*/\w*\.po', issue['title'])
```

Comment:
swap those to lines, and simply use `if yes:` to avoid calling re.search twice?

Reply:
Too sad because of backward compatibility we'll not use a walrus here :(

Reply:
See, this is why I love pair programming and code reviews. I didn't think of it...

Reply:
> Too sad because of backward compatibility we'll not use a walrus here :(

Yeah actually I was using it and then thought of @deronnax's future (?) PR that will backport to 3.5